src/Showpad/Client.php

```diff
@@ -222,15 +222,15 @@ public function assetsTagsList($assetId, $limit = 25, $offset = 0)
      *
      * @return array
      */
-    public function tagsList($limit = 25, $offset = 0)
+    public function tagsList($name, $limit = 25, $offset = 0)
```

Comment:
Thanks for changing this. I cannot merge yet, because this will break backwards compatibility with existing clients of this library. Also, name is optional so it could be added as last parameter instead of first, which would solve the BC problem.